app/views/avo/fields/common/_status.html.erb

```diff
@@ -0,0 +1,43 @@
+<%
+  classes = "inline-block "
+  classes += "text-red-700" if status == 'fail'
+%>
+<div class="flex flex-shrink-0">
+    <div class="<%= classes %>"><%= label %></div>
+    <% if status == 'load' %>
```

Comment:
Could we do `loading` instead of `load` and `failed` instead of `fail`? I know we had it like that before, but I think it makes more sense like this.